app/views/devise/passwords/new.html.erb

```diff
@@ -14,7 +14,10 @@
 <%= validated_form_for(@password_reset_email_form,
                        url: user_password_path,
                        html: { autocomplete: 'off', method: :post, role: 'form' }) do |f| %>
-  <%= f.input :email, required: true, input_html: { aria: { invalid: false, describedby: 'email-description' } } %>
+  <%= f.input :email,
+              required: true,
+              input_html: { autocorrect: 'off',
```

Comment:
can we add a regression check for this in `spec/views/devise/passwords/new.html.erb_spec.rb` to make sure we don't accidentally remove it in the future?